### PR TITLE
AinC ECS, TLC & Magnum [Ready for Review]

### PR DESCRIPTION
**Pull request by Tonnie15**

### CODE REVIEW

**Comment by Tonnie15** [comment]
ECS 
Hidden camo selections for "setObjectTexture":
0 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_OuterHaul_co.paa"
1 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_UpperParts_co.paa"
2 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_Tail_co.paa"
4 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_SideDoors_co.paa"
5 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_CargoDoors_co.paa"

ECS 
Hidden camo selections for "setObjectMaterial":
0 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_Gloss"
1 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_Gloss"
2 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_Gloss"
4 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_Gloss"
5 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_Gloss"

**Comment by Tonnie15** [comment]
TLC
Hidden camo selections for "setObjectTexture":
0 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_TLC_ainc_co.paa"

**Comment by Tonnie15** [comment]
Magnum Truck
Hidden camo selections for "setObjectTexture":
0 = "\UB_SND_TEX\textures\skins\ECS_AinC\UB_Custom_magnum_AinC_co.paa"

**Comment by arrproductions** [comment]
RVMAT is the extension not rvmap?

File: UB_SND_TEX/textures/skins/yo.sqf [hunk 1]
```diff
@@ -0,0 +1 @@
+1
```
Comment (fresqo) [inline UB_SND_TEX/textures/skins/yo.sqf]: What is this?

**Pull request merged**